server/src/project/_utils/transform-milestones.ts

```diff
@@ -146,6 +146,21 @@ function applyDisplayDate(milestone, project) {
   if (milestone._dcp_milestone_value === '780593bb-ecc2-e811-8156-1458d04d0698') milestone.display_date = milestone.dcp_actualenddate;
   if (milestone._dcp_milestone_value === '483beec4-dad0-e711-8116-1458d04e2fb8') milestone.display_date = milestone.dcp_actualenddate;
   if (milestone._dcp_milestone_value === '4a3beec4-dad0-e711-8116-1458d04e2fb8') milestone.display_date = milestone.dcp_actualenddate;
+  
+  /*
+  * The following is for task 13239
+  * The date displayed should be the CRM Review Meeting date for certain milestones if the milestone is "in progress" or "completed", and the meeting date is <= current date
+  * I have included a check to make sure that the field is not null to make sure we are not overwriting dates with nothing
+  * 2 is completed, 717170000 is in progress
```

Comment:
this part of the comments confuses me. Is this a typo? Did something accidentally get deleted?

Reply:
Oh, you are clarifying the statuscode. 

Can you add that in? 

Something like "Statuscode optionset: 
2 == completed
717170000 = "In progress"

Reply:
No, I just am not sure that there could not be an instance where all of the other criteria are met, but the CRM returns a null value for the reviewmeetingdate.  So I added a check to make sure it was not null, to ensure we wouldn't overwrite the date we want to use with a null value.

Reply:
Oh I meant can you clarify the comment?


Also.... even if there is already data in `milestone.display_date`,  how do we know it's not a misleading date? Since we want to make sure all dates displayed are "dcp_reviewmeetingdate", wouldn't it be better to assign "null" if `dcp_reviewmeetingdate` is null?